Formula/tbb.rb

```diff
@@ -20,6 +20,8 @@ class Tbb < Formula
   depends_on :python if MacOS.version <= :snow_leopard
   depends_on "swig" => :build
 
+  patch :DATA
```

Comment:
Please add a link to the upstream patch, if possible proof that it has been accepted by upstream